docs/tutorials/work_with_lsdb.ipynb

```diff
@@ -7,7 +7,9 @@
    "source": [
```

Comment:
<p>I added aiohttp as a dependency (perhaps unwisely?) for Nested-Dask (it was needed for a unit test), so it may not need special attention here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-dask/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
It doesn't work w/o aiohttp =(
https://readthedocs.org/projects/nested-dask/builds/24554523/

Reply:
Hmm, weird. Should be fine to keep it as an import here then, sorry for the misdirect!